### PR TITLE
DCS-1425 Tidying up swagger doc and responses

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/ArrivalsResource.kt [hunk 1]
```diff
@@ -77,7 +77,7 @@ class ArrivalsResource(
       ),
     ]
   )
-  @GetMapping(path = ["/incoming-moves/{agencyId}", "/prisons/{agencyId}/arrivals"])
+  @GetMapping(path = [ "/prisons/{agencyId}/arrivals"])
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/ArrivalsResource.kt:80]: `/incoming-moves` was the legacy version

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/PrisonResource.kt [hunk 2]
```diff
@@ -68,7 +68,7 @@ class PrisonResource(
   )
 
   @GetMapping(
-    value = ["/prison/prisoner/{prisonNumber}/image", "/prisoner/{prisonNumber}/image"],
+    value = ["/prison/prisoner/{prisonNumber}/image", "/prisoners/{prisonNumber}/image"],
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/PrisonResource.kt:71]: think we should move WPIP UI to `/prisoners/{prisonNumber}/image` and remove `/prison/prisoner/{prisonNumber}/image`

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/TransfersResource.kt [hunk 3]
```diff
@@ -137,14 +138,8 @@ class TransfersResource(
     security = [SecurityRequirement(name = "ROLE_TRANSFER_PRISONER", scopes = ["write"])],
     responses = [
       ApiResponse(
-        responseCode = "200",
+        responseCode = "204",
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/TransfersResource.kt:141]: this returns no content so now being explicit

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**